src/ROUTES.js

```diff
@@ -74,7 +74,8 @@ export default {
     IOU_DETAILS_ADD_DEBIT_CARD: `${IOU_DETAILS}/add-debit-card`,
     IOU_DETAILS_ENABLE_PAYMENTS: `${IOU_DETAILS}/enable-payments`,
     IOU_DETAILS_WITH_IOU_REPORT_ID: `${IOU_DETAILS}/:chatReportID/:iouReportID/`,
-    getIouDetailsRoute: (chatReportID, iouReportID) => `iou/details/${chatReportID}/${iouReportID}`,
+    getIouDetailsRoute: (chatReportID, iouReportID) => `iou/details/${chatReportID}/${iouReportID}/`,
+    getIouDetailsWithActionRoute: (chatReportID, iouReportID, iouReportAction) => `iou/details/${chatReportID}/${iouReportID}/${encodeURIComponent(iouReportAction)}/`,
```

Comment:
I am not sure if I understand, what if you have multiple pending actions? It seems like this page https://user-images.githubusercontent.com/242568/208656289-9d902401-af0b-42c8-8339-35fd93ce6de3.png already knows that the action is pending, we should wrap the IOUPreview.

This is nested in the IOUDetailsPage https://github.com/Expensify/App/blob/2585ceffcbd112e7e1207b7df8161e6132fa0b3b/src/components/ReportTransaction.js#L58-L72, we could wrap [this IOUPreview](https://github.com/Expensify/App/blob/2585ceffcbd112e7e1207b7df8161e6132fa0b3b/src/pages/iou/IOUDetailsModal.js#L147-L151) in offline with Feedback as well and get the action in there.

Thoughts?

Reply:
need to look at this, so maybe, but I also need to check if the Chat Report action Items are not affected.
TBH I think one of them is getting `opacity: 0.5` twice

Reply:
Not sure, what I mean is that in the IOUDetailsPage we already have an information that the reportAction is pending, and also we have this information for the preview in another view so I think there should be some other way, I know this is not very helpful haha, I would have to explore more on my own and dont want to duplicate work